arcgis-ios-sdk-samples/Scenes/Scene symbols/SceneSymbolsViewController.swift

```diff
@@ -70,7 +70,7 @@ class SceneSymbolsViewController: UIViewController {
             let point = AGSPoint(x: x + 0.01 * Double(offset), y: y, z: z, spatialReference: .wgs84())
             return AGSGraphic(geometry: point, symbol: symbol)
         }
-        
```

Comment:
It looks like one of the changes went away, but GitHub is still saying that this file was modified. Could you please get rid of this one as well?